tests/metadata.py

```diff
@@ -1,13 +1,30 @@
 from pgcli.packages.parseutils.meta import FunctionMetadata, ForeignKey
 from prompt_toolkit.completion import Completion
 from functools import partial
+from itertools import product
+from prompt_toolkit.document import Document
+from mock import Mock
+import pytest
 
 escape = lambda name: ('"' + name + '"' if not name.islower() or name in (
     'select', 'insert') else name)
 
+parametrize = pytest.mark.parametrize
```

Comment:
I'd actually struggled a bit with trying to write it as an import, but all I ended up with was this:
```
    from pytest.mark import parametrize
E   ImportError: No module named mark
```